runtime/common/Cargo.toml

```diff
@@ -8,6 +8,7 @@ homepage = "https://centrifuge.io/"
 repository = "https://github.com/centrifuge/centrifuge-chain"
 
 [dependencies]
+hex = { version = "0.4.3", default_features = false }
```

Comment:
I guess this can be removed now that it lives in types.
But no blocker.

Reply:
Thank you for the catch 💯